### PR TITLE
Fix naming convention

### PR DESCRIPTION
**Pull request by rgrinberg**
Prefer snake_case for our identifiers

### CODE REVIEW

File: src/ocaml_lsp.ml [hunk 1]
```diff
@@ -40,4 +40,4 @@ let of_initialize_result (t : LanguageClient.InitializeResult.t) =
 
 let interfaceSpecificLangId t = t.interfaceSpecificLangId
 
-let canHandleSwitchImplIntf t = t.handleSwitchImplIntf
+let can_handle_switch_impl_intf t = t.handleSwitchImplIntf
```
Comment (mnxn) [inline src/ocaml_lsp.ml:43]: Both of these should be snake_case

Comment (rgrinberg) [reply src/ocaml_lsp.ml:43]: For the field names, perahps we should keep them to match the identifiers from capabilities payload?

Comment (mnxn) [reply src/ocaml_lsp.ml:43]: I mean that `let interfaceSpecificLangId` should be snake_case if `let can_handle_switch_impl_intf` is. The field names can stay the same.

Comment (rgrinberg) [reply src/ocaml_lsp.ml:43]: OK. I added a `has` prefix to match the `can` prefix of the other function.

**Review by mnxn — APPROVED** [review]

**Pull request merged**